### PR TITLE
EVG-19827 Add logging to refresh in commit queue

### PR DESCRIPTION
**Pull request by bynn**
EVG-19827

### Description
add logging to see how often this happens
will be deleted after investigation

### CODE REVIEW

File: rest/data/commit_queue.go [hunk 1]
```diff
@@ -397,6 +397,13 @@ func getPRAndCheckMergeable(ctx context.Context, env evergreen.Environment, sc C
 				"pr":      info.PR,
 			}))
 		} else if p != nil && p.Version != "" {
+			grip.Debug(message.Fields{
+				"message": "calling refresh from commit queue",
```
Comment (ablack12) [inline rest/data/commit_queue.go:401]: might also be interesting to log if this actually changes this status, so maybe logging after 412 instead and logging mergeableState

**Review by ablack12 — APPROVED** [review]
LGTM with thought

**Comment by ablack12** [comment]
Ticket doesn't have a description but I imagine we intend to look at the logging and remove it before closing the ticket?

File: rest/data/commit_queue.go (hunk 1 above)
Comment (bynn) [reply rest/data/commit_queue.go]: moved and printing new mergable state!

**Pull request merged**